### DISCUSSION TITLE
Window Placement on Startup

### DISCUSSION THREAD

**Discussion by rontarrant** [godotengine/godot-proposals#5095] [Editor]
There's a hack that resizes Godot's editor window so its not maximized on start-up, but it only works on a per-project basis.

What would be nice is to rework the Godot editor so it always remembers these things:

- window size, and
- window placement.

And it would be nice if we didn't have to keep setting it up over and over every time we start a new project, just set it once and forget it.

It's a small thing, but Oh, man! It would help keep me sane.

**Comment by Calinou** [comment]
This is being worked on in https://github.com/godotengine/godot/pull/32056, but it requires https://github.com/godotengine/godot-proposals/issues/3145 to be implemented first to be done properly.

**Comment by rontarrant** [comment]
Apples and oranges. You're talking about apps built in Godot whereas I'm
talking about Godot itself... the Godot editor window always opens full
screen and on a 2k or 4k monitor, it's overkill. Try it and you'll see what
I'm talking about.
